chessjs.md

```diff
@@ -0,0 +1,9 @@
+[Chess.js](https://github.com/jhlywa/chess.js) is a headless chess logic library that you can use if you wanted to build a chess game in javascript, and you already had the UI ready.
```

Comment:
move to `projects/javascript/chessjs.md`

Reply:
Done.